src/utils/common.ts

```diff
@@ -421,3 +423,11 @@ export function checkIfSuperheroCallbackUrl(query: LocationQuery) {
     (value) => value && (value as string).startsWith(slicedAggregatorUrl),
   );
 }
+
+export function isCoin(contractId: IToken['contractId']): boolean {
+  return PROTOCOL_LIST.some(
+    (protocol) => ProtocolAdapterFactory
+      .getAdapter(protocol)
+      .getDefaultAssetContractId() === contractId,
+  );
+}
```

Comment:
Maybe the `getDefaultCoin` method could return the flag that would determine if it's coin?

Reply:
That would mean calling the `getDefaultCoin` method with an extra `contractId` argument
We would also be calling `getDefaultCoin` without wanting to get the default coin. I think it makes sense to keep this functionality separate